docs/configuration/configuration-file.mdx

```diff
@@ -69,6 +72,42 @@ You may also import plugins and extends their configurations.
 
 Read more about the [lint configuration section](../guides/lint.md).
 
+
+### region
```

Comment:
I moved everything about the region setting to this config file, since it can be configured here as well. Then we can just link to this from the commands pages instead of repeating the same content.

However, I'm not sure the info here is 100% correct so it would be great if @adamaltman or @RomanHotsiy could take another look. Thanks! :)

Reply:
@skadinna tnx a lot